src/Avalonia.Controls/TreeView.cs

```diff
@@ -132,6 +128,7 @@ public SelectionMode SelectionMode
         /// <summary>
         /// Gets or sets the selected items.
         /// </summary>
+        [AllowNull]
         public IList SelectedItems
```

Comment:
Why is attribute used here instead of "?" ?

Reply:
To match the existing accessor implementations. The getter should imo definitely stay not null as it's very convenient: callers can add/remove items to the collection directly. It's also the backing property for the `SelectedItemsProperty`. The setter already accepted null and I reflected that by using `AllowNull` to annotate  the setter without affecting the getter.

Reply:
So, I suppose setting AllowNull on the whole property affects only setter, while not affecting getter. Right?
Like an equivalent of `{ get {} [AllowNull] set {} }`

Reply:
Yes, `AllowNull` isn't valid on methods and only affects input, so the setter. Not a very common pattern, but you can find it in [some places in the runtime](https://github.com/dotnet/runtime/blob/d913b94d041e192ab2f389ecae6b972da2094560/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs#L61). `MaybeNull` only affects the getter.